requirements.txt

```diff
@@ -38,6 +38,7 @@ pandas~=2.2.3
 paramiko~=3.4.1
 plyer~=2.0.0
 psycopg2-binary~=2.9.9
+pyarrow~=18.0.0
```

Comment:
Perhaps also add pyarrow-stubs inside requirements-dev.txt. I did it inside my old PR probably to shush mypy issues